src/blocks/product-card.css

```
@@ -0,0 +1,95 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 16px;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+}
+
+.product-card__picture {
+  margin-top: 16px;
+  align-self: center;
+
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__description {
+  margin-top: 40px;
+
+  color: var(--main-font-color);
+  line-height: 18px;
+}
+
+.product-card__product-code {
+  align-self: start;
+  margin-top: 4px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-font-color);
+
+}
+
+.product-card__rating {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.rating__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--main-font-color);
+}
+
+.product-card__price {
+  margin-top: 24px;
+
+  display: flex;
+  justify-content: space-between;
+
+}
+
+.price__title {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--secondary-font-color);
+}
+
+.price__item {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-font-color);
+}
+
+.product-card__button {
+  margin-top: 16px;
+  width: 164px;
+  height: 38px;
```

do not hardcode width and height, for height use vertical paddings
```suggestion
  width: 100%;
```